### PR TITLE
(docs): change progress-circle to progress-spinner

### PR DESCRIPTION
**Pull request by alexisbmills**
Fix broken link and change docs to reflect renaming progress-circle to progress-spinner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ High level items planned for December 2016:
 | list             |                           Available | [README][8]  |   [#107][0107] |
 | grid-list        |                           Available | [README][9]  |              - |
 | icon             |                           Available | [README][10] |              - |
-| progress-circle  |                           Available | [README][11] |              - |
+| progress-spinner  |                           Available | [README][11] |              - |
```
Comment (DevVersion) [inline README.md]: Can you try to align the pipes in this file?

Comment (alexisbmills) [inline README.md]: Aha, no problem.

Comment (DevVersion) [inline README.md]: Thanks

**Comment by alexisbmills** [comment]
I signed the CLA prior to the pull request. Does it perhaps take some time to register?

**Comment by DevVersion** [comment]
@alexisbmills The problem is that the commits aren't signed with the email you're using for GitHub and the CLA.

**Comment by jelbourn** [comment]
LGTM, thanks!

**Pull request merged**